distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -36,8 +36,8 @@ type Number
          Adding 10 and 15.
 
              10 + 15
-    + : Number -> Number
-    + that = @Builtin_Method "Integer.+"
+    + : Number -> Number -> Number
```

Comment:
Why did you make this change only once? Surely either this signature is now incorrect, or the one for `*` is

Reply:
Omission, thanks for catching it. I started doing it at some point but then reverted to not including type info about self.
It's actually feels like a separate discussion because maybe we should, in general.